ui/admin/app/services/feature-edition.js

```diff
@@ -3,7 +3,7 @@ import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 import config from 'admin/config/environment';
 
-const { selectedEdition, featureEditions } = config;
+console.log(config.features.licensedFeatures);
```

Comment:
None blocking comment: this console, is for development usage? I understand it as it will console log the list of licensed features?

Reply:
Good catch this should be removed.